openwisp_firmware_upgrader/tests/test_api.py

```diff
@@ -757,9 +762,17 @@ def test_firmware_download(self):
         with open(self.FAKE_IMAGE_PATH, 'rb') as f:
             content = f.read()
         url = reverse('upgrader:api_firmware_download', args=[image.build.pk, image.pk])
-        with self.assertNumQueries(6):
-            r = self.client.get(url)
-        self.assertEqual(r.getvalue(), content)
+        with self.subTest("Test as operator"):
+            self._make_operator_org_manager()
+            with self.assertNumQueries(6):
+                response = self.client.get(url)
+            self.assertEqual(response.getvalue(), content)
+        with self.subTest("Test as superuser"):
```

Comment:
Superuser is saved 3 queries tat happen in `user.is_manager()` usage.

Reply:
probably because `self.object.build.category.organization` yuck